mlkem/kem.h

```diff
@@ -35,6 +36,77 @@
 #define crypto_kem_dec MLK_NAMESPACE_K(dec)
 #endif
 
+typedef struct
+{
+  mlk_indcpa_secret_key indcpa_sk;
+  mlk_indcpa_public_key indcpa_pk;
+  uint8_t seed[MLKEM_SYMBYTES];
+  uint8_t z[MLKEM_SYMBYTES];
+  uint8_t hpk[MLKEM_SYMBYTES];
+} mlk_secret_key;
+
+typedef struct
+{
+  mlk_indcpa_public_key indcpa_pk;
+  uint8_t hpk[MLKEM_SYMBYTES];
+} mlk_public_key;
+
+
+MLK_EXTERNAL_API
```

Comment:
MLK_MUST_CHECK_RETURN_VALUE is missing in a few places here?

Reply:
yes.